server/cluster/cluster.go

```diff
@@ -1068,6 +1076,7 @@ func (c *RaftCluster) BuryStore(storeID uint64, forceBury bool) error {
 	if err == nil {
 		// clean up the residual information.
 		c.RemoveStoreLimit(storeID)
+		c.resetProgress(storeID, store.GetAddress(), removingAction)
```

Comment:
What about we forcibly set the store state back to the `Up` state? Do we also need to reset the progress?